bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/model/SomfyTahomaCommand.java

```diff
@@ -27,7 +28,7 @@
 
     int type;
     String name;
-    ArrayList<Object> parameters = new ArrayList<Object>();
+    List<Object> parameters = new ArrayList<Object>();
```

Comment:
```
 List<Object> parameters = new ArrayList<>();
```

That is out of scope of this PR, but: Is it really necessary that this is `Object`? In `getPercentParameters` it is assumed that all list-members can be casted to `Double`.

Reply:
I have got rid of the obsolete SomfyTahomaCommand. thanks